### PR TITLE
TMA-807: log to splunk from K8S

### PR DESCRIPTION
**Pull request by kubamahnert**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1199/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1200/ : SUCCESS in 8s

File: lib/gooddata/rest/connection.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (panjan) [inline lib/gooddata/rest/connection.rb]: Empty lines added.

File: lib/gooddata/core/logging.rb [hunk 2]
```diff
@@ -21,7 +21,7 @@ module GoodData
 
   DEFAULT_SPLUNKLOG_LEVEL = Logger::INFO
   DEFAULT_SPLUNKLOG_OUTPUT = STDERR
-  DEFAULT_SPLUNKLOGGER_CLASS = SplunkLogger
```
Comment (panjan) [inline lib/gooddata/core/logging.rb]: DEFAULT_SPLUNKLOGGER_CLASS seems to be unused

**Comment by kubamahnert** [comment]
https://splunk-sh4.na.intgdc.com/en-GB/app/search/search?q=search%20index%3D*%20source%3D%2Fmnt%2Flog%2Fgdc-ruby&display.page.search.mode=fast&dispatch.sample_ratio=1&earliest=1550507168.000&latest=1550507168.001&sid=1550510208.153199.splunk-sh4.int.na.intgdc.com

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1201/ : SUCCESS in 17s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1202/ : SUCCESS in 10s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1203/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/215/ : SUCCESS in 23s

**Pull request merged**